### PR TITLE
mk/USA-152: Create a new rule to prevent datetime_now use

### PR DESCRIPTION
**Pull request by meltemkenis**
https://plentific.atlassian.net/browse/USA-152

### CODE REVIEW

File: datetime_now.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+from pylint.checkers import BaseChecker
+
+
+class DateTimeNowChecker(BaseChecker):
+
+    __implements__ = IAstroidChecker  # pylint: disable=F0220
+
+    name = 'datetime_now'
+    msgs = {
+        'R0402': ('Do not use datetime.now() %s',
```
Comment (jakubwilkowski) [inline datetime_now.py]: Would be good to tell what to use instead, so django.uilts...timezone.now()

**Review by jakubwilkowski — APPROVED** [review]
In general it looks good, just please improve the comment. 

I don't have that much exp with pylint plugins, so what I can suggest is a couple of test to see if this plugins works as intended:

#### 1
from datetime import datetime
datetime.now() <-- fail

#### 2
import datetime
datetime.datetime.now() <-- fail

#### 3
from django.utils... import timezone
timezone.now() <-- ok

#### 4

import datetime
now = datetime.datetime.now
now() <-- fail

**Comment by daberg** [comment]
Hi @jakubwilkowski @meltemkenis I worked on some fixes for the plugin but I can't push them since I don't have write perms to the repo. Can you please give  them to me? Thanks! (Not sure how it's done btw)

**Review by emredogan7 — APPROVED** [review]
@daberg can you please share a few test results as a PR comment?

**Review by jakubwilkowski — APPROVED** [review]
When we'll release this, please post a message to eng channel that we're making this change, why we're adding it and what's the intended approach

**Comment by daberg** [comment]
## Test cases

```
datetime.now()

datetime.datetime.now()

test = 'test' + datetime.now()

test = datetime.now

((datetime)).now()

def mytest():
    return 'test' + datetime.now()
```
(6 warnings raised)

---

```
timezone.datetime.now()

import datetime

from datetime import datetime

now.datetime()

datetime.test()

datetime.datetime.test()

timezone.datetime.now()

def datetime(datetime, now):
    datetime(now)
    return now
```
(no warnings)

**Pull request merged**